pkg/database/orm/orm.go

```diff
@@ -73,19 +73,39 @@ func (o *ORM) Initialize() error {
 			o.config.DbUser,
 			o.config.DbPassword,
 			o.config.ConnectionParams)
+		dbDialector = postgres.Open(o.connectionStr)
 	}
 
-	opened, err := gorm.Open(databaseType, o.connectionStr)
+	opened, err := gorm.Open(dbDialector, &gorm.Config{
+		SkipDefaultTransaction:                   false,
+		NamingStrategy:                           nil,
+		FullSaveAssociations:                     false,
+		Logger:                                   nil,
```

Comment:
wouldn't it better to pass a logger?

Reply:
yes it would but need to implement the new interface.
I can add it as new issue for now if it is acceptable

Reply:
Mmmm yeah but I would say it's quite an important issue so if you're thinking about doing it soon I'm ok.

Reply:
Done by @Tibo-lg thanks